about.html

```diff
@@ -0,0 +1,225 @@
+        </div>
+      </section>
+    </nav>
+
+    <main class="main_about">
+      <section class="about_section">
+        <h2>
+          <em> We share knowledge to transform the world through learning</em>
+        </h2>
+      </section>
+      <section class="vision">
+        <header class="vision_header">
+          <h2>Our Vision and mission</h2>
+        </header>
+        <article class="vision_article">
+          <p class="vision_paragraph">
+            Lorem ipsum, dolor sit amet consectetur adipisicing elit. Libero
+            optio reiciendis pariatur quas impedit corporis cupiditate sunt fuga
+            voluptatibus fugit earum, iusto veniam ipsa culpa explicabo! Nam hic
+            a repellat.
+          </p>
+          <ul class="vision_lists">
+            <li class="vision_list">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Sapiente,
+              error.
+            </li>
+            <li class="vision_list">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Sapiente,
+              error.
+            </li>
+            <li class="vision_list">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Sapiente,
+              error.
+            </li>
+          </ul>
+        </article>
+      </section>
+    </main>
+    <section class="data_section">
+      <header class="data_section_header">
+        <h2>We Just Keep Growing</h2>
+        <div class="data_underline"></div>
+      </header>
+      <article class="data_article">
+        <h4 class="data_description">Our universal community and our course catalog get bigger every day. Check out our latest numbers as of August 2022.</h4>
+        <div class="all_data">
+        <div class="data">
+          <h2>70M+</h2>
+          <p>Learners</p>
+        </div>
+        <div class="data">
+          <h2>104K+</h2>
+          <p>Instructors</p>
+        </div>
+        <div class="data">
+          <h2>300K+</h2>
+          <p>Courses</p>
+        </div>
+        <div class="data">
+          <h2>960M+</h2>
+          <p>Course Enrollment</p>
+        </div>
+        </div>
+      </article>
+
+    </section>
+    
+    <section>
+      <header class="our_story_header">
+        <h2>Our Story</h2>
+      </header>
+      <article class="our_story_article">
+        <p class="story_paragraph">
+          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Eius
+          eveniet similique nesciunt accusamus est minus illo culpa
+          temporibus? Obcaecati mollitia consequatur totam architecto
+          officiis, cum nostrum voluptates maxime laboriosam possimus
+          assumenda eos est neque deleniti adipisci ratione eaque hic incidunt
+          libero esse vero. Possimus ipsa ratione sequi deleniti blanditiis
+          libero? Lorem ipsum dolor sit amet consectetur, adipisicing elit. Excepturi cum reiciendis quasi in voluptas delectus rerum quaerat? Debitis, magnam quaerat!
```

Comment:
- [ ] Kindly make your design match the one in the Behance by adding the following sections like in the image.
![image](https://user-images.githubusercontent.com/53402600/190259989-716b3121-6b4a-4190-bc7f-798193f6312d.png)

Reply:
@omarsalem7, I designed a Web Development website  so I actually think this about page design doesn't match web development platform design. That's why I did it that way

Reply:
Hi @Fayob, I agree with you but you should follow the guidelines which you need to follow, and for my work I have guidelines also I need to check if you follow them. I suggest you can add a part for past summits, you can add past two images for students who take a course online or offline. Moreover, you can adjust the content as you want but follow the same design.